client/fvAEPg_service.go

```diff
@@ -465,6 +464,39 @@ func (sm *ServiceManager) CreateRelationfvRsNodeAttFromApplicationEPG(parentDn,
 	return nil
 }
 
+func (sm *ServiceManager) CreateRelationfvRsNodeAtt(parentDn, encap, mode, description, deploymentImmediacy, tDn string) error {
+	dn := fmt.Sprintf("%s/rsnodeAtt-[%s]", parentDn, tDn)
+	containerJSON := []byte(fmt.Sprintf(`{
```

Comment:
Do we need `"annotation":"orchestrator:terraform"` to be added in the statically to payload like we do in the other functions or is this handled differently? Another option would be to pass to method, `"annotation": "%s"` and dynamically fill in the payload.